10.9-libcxx/stable/main/finkinfo/text/docutils-0181-py.info

```diff
@@ -0,0 +1,111 @@
+Info2: <<
+Package: docutils-py%type_pkg[python]
+Type: python (2.7 3.4 3.5 3.6 3.7 3.8)
+Version: 0.18.1
+Revision: 1
+Distribution: <<
+	(%type_pkg[python] = 34 ) 10.9,
+	(%type_pkg[python] = 34 ) 10.10,
+	(%type_pkg[python] = 34 ) 10.11,
+	(%type_pkg[python] = 34 ) 10.12,
+	(%type_pkg[python] = 34 ) 10.13,
+	(%type_pkg[python] = 34 ) 10.14,
+	(%type_pkg[python] = 34 ) 10.14.5,
+	(%type_pkg[python] = 34 ) 10.15,
+	(%type_pkg[python] = 35 ) 10.9,
+	(%type_pkg[python] = 35 ) 10.10,
+	(%type_pkg[python] = 35 ) 10.11,
+	(%type_pkg[python] = 35 ) 10.12,
+	(%type_pkg[python] = 35 ) 10.13,
+	(%type_pkg[python] = 35 ) 10.14,
+	(%type_pkg[python] = 35 ) 10.14.5,
+	(%type_pkg[python] = 35 ) 10.15,
+	(%type_pkg[python] = 36 ) 10.9,
+	(%type_pkg[python] = 36 ) 10.10,
+	(%type_pkg[python] = 36 ) 10.11,
+	(%type_pkg[python] = 36 ) 10.12,
+	(%type_pkg[python] = 36 ) 10.13,
+	(%type_pkg[python] = 36 ) 10.14,
+	(%type_pkg[python] = 36 ) 10.14.5,
+	(%type_pkg[python] = 36 ) 10.15
+<<
+Source: https://files.pythonhosted.org/packages/source/d/docutils/docutils-%v.tar.gz
+Source-Checksum: SHA256(679987caf361a7539d76e584cbeddc311e3aee937877c87346f31debc63e9d06)
+
+Depends: python%type_pkg[python]
+
+Conflicts: docutils (<= 0.4-2)
+Replaces: docutils (<= 0.4-2)
+Provides: docutils
+
+CompileScript: <<
+  %p/bin/python%type_raw[python] setup.py build
+<<
+
+InstallScript: <<
+#!/bin/sh -ev
+%p/bin/python%type_raw[python] setup.py install --root %d
+
+for i in *.txt ; do
+  htmldoc=${i%%txt}html
+  echo "Making $htmldoc"
+  PYTHONPATH=.:docutils/utils tools/rst2html.py $i $htmldoc
+done
+
+cd docs
+for i in `find . -name '*.txt'` ; do
+  destdir="%i/share/doc/%n/html/"${i%%/*}
+  htmldoc=${i%%txt}html
+  echo "Making $htmldoc"
+  PYTHONPATH=..:../docutils/utils ../tools/rst2html.py $i $htmldoc
+  mkdir -p "$destdir"
+  cp $htmldoc $destdir
+done
+
+cd %i/bin
+for i in rst2* rstpep2html.py; do
```

Comment:
```suggestion
for i in docutils rst2* rstpep2html.py; do
```

Reply:
Thanks!   Ah, that is python version not a spoken language version that is the problem.
should the line
`cp  %b/tools/rstpep2html.py %i/bin`
also be python versioned?  Something like this to be consistent?
`cp %b/tools/rstpep2html.py %i/bin/rstpep2html-%type_raw[python].py`

Reply:
Ah yes, I missed there are two distribution versions. Since it's also installed into `%i/bin`, it should need to be python-versioned or validation should fail as well.

TeXLive would be a quite heavy dependency, though probably a recommendation, as one of the principal outputs is to LaTeX format. It would seem like it has its own TeX parser if it gets that far though, just having problems with some mathtex commands.

Reply:
yes, TeXLive would be a heavy lift.  But if it has its own TeX parser why is the building of the documentation failing like that?  If I can get TeXLive built and installed, I will see if the errors go away.  Probably but something in the build notes that the mathematics documentation doesn't build unless you have it installed?

But still baffled by inability to get TeXLive-texmf to install.